doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
+| committer        | The user that committed this revision, if defined   | User          |             | READ                 |
```

Comment:
I'm not even sure the repository backend would be consistent with the differing role of `author` and  `committer`, thus author is fine for me.